### ISSUE TITLE
tsd should be devDependency and not a dependency

### ISSUE THREAD

**Issue by IgorMinar** [angular/ts2dart#159]
Since tsd is now a `dependency` in package.json all projects that depend on ts2dart have to install it. tsd and it's dependencies are quite sizable so that's far from ideal.

Making tsd a devDependency resolves this issue because devDependencies are not part of the transitive closure of dependencies in a project that uses ts2dart.

Note: I know that this complicates the publish process, but maybe replacing that with a shell script or some task runner would be a better choice.
